certbot/tests/util_test.py

```diff
@@ -52,26 +53,42 @@ def _call(cls, exe):
         from certbot.util import exe_exists
         return exe_exists(exe)
 
-    @mock.patch("certbot.util.os.path.isfile")
```

Comment:
I have the same high level comments about testing here as I do above. I personally think these tests should be moved to `filesystem_test.py` and we can keep simple tests here that just mock out `filesystem.is_executable` if we want.

Reply:
Done.